QuizAPI/DTOs/QuestionDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace QuizAPI.DTOs
+{
+	public class QuestionDTO
+	{
+		private string? question;
+		private string? answer;
+		private string? difficulty;
+		private string? category;
+		private string? status;
```

Comment:
Do you guys think it's a good idea to add status in the question DTO? It's not something a user should be able to change or interact with.

Reply:
Didn't we discuss that the status is for the admin to see and to see which questions have been approved and so on?